conn.go

```diff
@@ -1151,7 +1157,24 @@ func (st *stmt) exec(v []driver.Value) {
 			w.bytes(b)
 		}
 	}
-	w.int16(0)
+
+	// result-column format codes
+	var binary bool
+	for i, o := range st.rowTyps {
+		if o == oid.T_bytea {
+			binary = true
+			st.rowFmts[i] = formatBinary
+		}
```

Comment:
Probably.  But I don't think the first version has to do it.  Let's see what the numbers say first.